src/package.xml

```diff
@@ -462,6 +464,12 @@
         <name>ListView</name>
     </types>
     <types>
+        <members>Course_Enrollment__c.Default</members>
+        <members>Course_Enrollment__c.Faculty</members>
+        <members>Course_Enrollment__c.Student</members>
```

Comment:
@xinnanlisf We'll want these to be removed from the build. SInce these are part of the unpackaged deployments.